buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -44,12 +44,15 @@
 public abstract class AbstractByteBuf extends ByteBuf {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(AbstractByteBuf.class);
     private static final String PROP_MODE = "io.netty.buffer.bytebuf.checkAccessible";
+    private static final String BOUNDS_MODE = "io.netty.buffer.bytebuf.checkBounds";
```

Comment:
- `PROP_MODE` to `PROP_CHECK_ACCESSIBLE` and `BOUNDS_PROP` to `PROP_CHECK_BOUNDS`?
- `buffer.bytebuf` sounds redundant to me although it is consistent with `PROP_MODE`. How about using `io.netty.buffer.checkBounds` and accepting both `io.netty.buffer.bytebuf.checkAccessible` and `io.netty.buffer.checkAccessible`?

Reply:
> PROP_MODE to PROP_CHECK_ACCESSIBLE and BOUNDS_PROP to PROP_CHECK_BOUNDS

:+1: 

> How about using io.netty.buffer.checkBounds

:+1: 

> and accepting both io.netty.buffer.bytebuf.checkAccessible and io.netty.buffer.checkAccessible?

Sounds good, may I add some log on deprecated uses of `io.netty.buffer.bytebuf.checkAccessible` too, wdyt?

Reply:
> may I add some log on deprecated uses of io.netty.buffer.bytebuf.checkAccessible too, wdyt?

👍